src/applications/personalization/dashboard/actions/debts.js

```diff
@@ -45,7 +45,7 @@ export const fetchDebts = () => async dispatch => {
       .filter(debt => debt.currentAr > 0)
       .map((debt, index) => ({ ...debt, id: index }));
     recordEvent({
-      event: `get-debts-successful`,
+      event: `api_call`,
       'error-key': `server error`,

```

Comment:
should this be set to `server error` if this is the event for success?

Reply:
lol no. Let me fix that